iota/resources/deps.cljs

```diff
@@ -0,0 +1,7 @@
+{:foreign-libs [{:file     "cljsjs/development/iota.inc.js",
```

Comment:
`deps.cljs` is already generated by `deps-cljs` task so this file is unnecessary,